views/profile.jade

```diff
@@ -0,0 +1,55 @@
+extends layout
+
+block content
+.container
+    .row.profile">
```

Comment:
What is this weird character at the end? i see it in several places in this file.